### PR TITLE
Cherry picks

### PR DESCRIPTION
**Pull request by mmason2-godaddy**
## Summary
Cherry picks

### CODE REVIEW

**Review by kbader-godaddy — APPROVED** [review]

**Review by jpina1-godaddy — DISMISSED** [review]

File: docs/generated-docs/lifecycle-graphs.md [hunk 1]
```diff
@@ -39,10 +39,16 @@ start -- exec --> servers;
 express -- exec --> serviceWorkerCacheKey;
 preboot -- exec --> start;
 start -- execWaterfall --> terminus;
+<<<<<<< HEAD
```
Comment (jpina1-godaddy) [inline docs/generated-docs/lifecycle-graphs.md:42]: Is this leftover from the merge?

File: docs/generated-docs/lifecycle-graphs.md [hunk 2]
```diff
@@ -39,10 +39,16 @@ start -- exec --> servers;
...
 initWebpack -- execSync --> webpack["webpack (deprecated)"];
 initWebpack -- execSync --> webpackChain["webpackChain (deprecated)"];
 initWebpack -- execWaterfallSync --> webpackConfig;
 createLogger -- exec --> winstonTransports;
+=======
```
Comment (jpina1-godaddy) [inline docs/generated-docs/lifecycle-graphs.md:47]: Same here and on line 51?

File: README.md [hunk 3]
```diff
@@ -156,6 +156,7 @@ Available plugins
 
 | Name                            | Version | Description                                                                |
 | ------------------------------- | ------- | -------------------------------------------------------------------------- |
+<<<<<<< HEAD
```
Comment (jpina1-godaddy) [inline README.md:159]: Merge artifact

File: README.md [hunk 4]
```diff
@@ -185,12 +186,46 @@ Available plugins
 | [@gasket/plugin-swagger]        | 6.46.7  | Generate and serve swagger docs                                            |
 | [@gasket/plugin-webpack]        | 6.46.2  | Adds webpack support to your application                                   |
 | [@gasket/plugin-workbox]        | 6.46.2  | Gasket Workbox Plugin                                                      |
+=======
```
Comment (jpina1-godaddy) [inline README.md:189]: This block too

File: README.md [hunk 5]
```diff
@@ -208,6 +243,24 @@ Supporting modules
 | /Users/agerard/dev/ghec/godaddy/gasket/packages/gasket-core |         |                                                                                      |
 | /Users/agerard/dev/ghec/godaddy/gasket/packages/iso-test    |         |                                                                                      |
 | [create-gasket-app]                                         | 6.46.7  | starter pack for creating a gasket app                                               |
+=======
```
Comment (jpina1-godaddy) [inline README.md:246]: left over merge stuff

**Review by jpina1-godaddy — CHANGES_REQUESTED** [review]

**Comment by mmason2-godaddy** [comment]
Good catch @jpina1-godaddy

**Review by jpina1-godaddy — APPROVED** [review]

**Pull request merged**